cypress/support/step_definitions/validations.js

```diff
@@ -433,6 +433,41 @@ Given(
   }
 );
 
+/**
+ * @module validations
+ * @function Interactions metrics collection process is '(initiated|stopped)'
+ * @description To start or stop listening to firebolt interactions in device by passing appropriate intent to designated handler
+ * @param {String} action - initiated or stopped
+ * @example
+ * Given Interactions metrics collection process is inititated
+ * Given Interactions metrics collection process is stopped
+ */
+Given(/Interactions metrics collection process is '(initiated|stopped)'/, (action) => {
+  if (
+    (action == CONSTANTS.INITIATED &&
+      UTILS.getEnvVariable(CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED, false) != true) ||
+    (action == CONSTANTS.STOPPED &&
+      UTILS.getEnvVariable(CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED) == true)
+  ) {
+    cy.startOrStopInteractionsService(action).then((response) => {
+      if (response) {
+        Cypress.env(
+          CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED,
+          action == CONSTANTS.INITIATED ? true : false
+        );
+      } else {
+        const message =
+          action == CONSTANTS.INITIATED
+            ? CONSTANTS.FAILED_TO_INITIATE_INTERACTIONS_SERVICE
+            : CONSTANTS.FAILED_TO_STOP_INTERACTIONS_SERVICE;
+        fireLog.assert(false, message);
+      }
+    });
+  } else {
+    cy.log(CONSTANTS.INTERACTIONS_SERVICE_ENABLED);
+  }
+});
+
```

Comment:
query : Is this glue code changes of FIRECERT-2190 ?

Reply:
not exactly, just added glue code in 2189, so can be used for testing bolt changes